editor/debugger/editor_debugger_tree.cpp

```diff
@@ -32,6 +32,7 @@
 
 #include "editor/editor_node.h"
 #include "editor/gui/editor_file_dialog.h"
+#include "editor/editor_string_names.h"
```

Comment:
Static checks also complain about alphabetical order
```suggestion
#include "editor/editor_string_names.h"
#include "editor/gui/editor_file_dialog.h"
```